Userland/Libraries/LibGfx/ImageFormats/JPEGXLLoader.cpp

```diff
@@ -1133,7 +1133,72 @@ static ErrorOr<TransformInfo> read_transform_info(LittleEndianInputBitStream& st
 ///
 
 /// Local abstractions to store the decoded image
-class Image {
+class ImageAPI {
```

Comment:
nit: "ImageAPI" is a bit of an unusual name. I'd probably name this `BlendedImage` or something like that.

Reply:
Done!